.npmignore

```diff
@@ -0,0 +1,11 @@
+# Tests
```

Comment:
you don't need this file at all since the root level package will never be published